.github/workflows/ci.yaml

```diff
@@ -34,12 +33,13 @@ jobs:
       - name: Setup Conda Environment
         uses: mamba-org/provision-with-micromamba@main
         with:
-          environment-file: devtools/conda-envs/test_env.yaml
+          environment-file: devtools/conda-envs/test_env_${{ matrix.toolkits }}.yaml
           channel-priority: "flexible"
           extra-specs: |
-            python=${{ matrix.cfg.python-version }}
+            python=${{ matrix.python-version }}
```

Comment:
I'm pretty sure you still want `cfg.` since that's still in the file above

```
      matrix:
        cfg:
          os: ["ubuntu-latest"]
          python-version: ["3.9", "3.10"]
          toolkits: ["rdkit_and_openeye", "rdkit"]

Reply:
Thanks! Any objection to me removing `cfg`? I think the toolkit CI did so some time ago.